src/nuxeo-cloud/what-is-nuxeo-cloud.md

```diff
@@ -0,0 +1,208 @@
+---
+title: What's Nuxeo Cloud
+review:
+    comment: ''
+    date: '19-03-2020'
+    status: ok
+labels:
+    - nuxeo-cloud
+tree_item_index: 100
+toc: true
+---
+
+Nuxeo Cloud is a fully hosted and managed offering providing our customers with setup, support and maintenance for a complete Nuxeo implementation.
```

Comment:
change to "Nuxeo Cloud is a hosted and managed offering that provides our customers with the setup, support and maintenance for a complete implementation of the Nuxeo Platform."